src/components/layers/layers.component.ts

```diff
@@ -9,43 +7,21 @@ import {
 })
 export class LayersComponent {
     @Input() current: number;
-    @Output() selected = new EventEmitter();
+    @Output() select = new EventEmitter();
 
-    @ViewChildren('baseButton,modButton,fnButton,mouseButton')
-    buttonsQueryList: QueryList<ElementRef>;
+    private buttons: string[];
 
-    private buttons: ElementRef[];
-    private selectedLayerIndex: number;
-
-    constructor(private renderer: Renderer) {
-        this.buttons = [];
-        this.selectedLayerIndex = 0;
-    }
-
-    ngOnChanges() {
-        if (this.buttons.length > 0 && this.current !== this.selectedLayerIndex) {
-            this.buttons.forEach((button: ElementRef) => {
-                this.renderer.setElementClass(button.nativeElement, 'btn-primary', false);
-            });
-            this.renderer.setElementClass(this.buttons[this.current].nativeElement, 'btn-primary', true);
-            this.selectedLayerIndex = 0;
-        }
+    constructor() {
+        this.buttons = ['Base', 'Mod', 'Fn', 'Mouse'];
+        this.current = 0;
     }
 
     selectLayer(index: number) {
-        if (index === this.selectedLayerIndex) {
```

Comment:
Please add this back, so that if you click on current button, you don't trigger the whole tree.